components/formats-gpl/src/loci/formats/in/LIFReader.java

```diff
@@ -2040,7 +2077,7 @@ private void translateImageNodes(Element imageNode, int i)
       if (StringUtils.isBlank(v)) {
         physicalLen = 0d;
       } else {
-        physicalLen = new Double(v.trim());
+        physicalLen = DataTools.parseDouble(v.trim());
```

Comment:
This seem to have broken https://web-proxy.openmicroscopy.org/east-ci/job/BIOFORMATS-test-folder/40072/ with some length values being off by a factor 100 .

Reply:
The breakage actually seems to be correct, and is a result of https://github.com/ome/ome-common-java/pull/30. In the process of updating configurations on https://github.com/openmicroscopy/data_repo_config/pull/342 with supporting screenshots from LAS X.